data/feature-flags.ts

```diff
@@ -12,7 +12,8 @@ export const featureFlags = [
       '0x5cce', // saulmc.lens
       '0x014309', // iambhavya.lens
       '0x8539', // petermdenton.lens
-      '0x013f9d' // shanemac.lens
+      '0x013f9d', // shanemac.lens
+      '0x4cc3' // iambhavya2.lens
```

Comment:
Is this for testnet? If so, everyone should be enabled by default in testnet. Only mainnet profiles need to be allowlisted here:

https://github.com/lensterxyz/lenster/blob/bf58e32fb178d8e6be3d6035e804fbb77d1c001f/src/lib/isFeatureEnabled.ts#L7